documentation/src/main/asciidoc/chapter-2-set-up.asciidoc

```diff
@@ -254,6 +254,18 @@ Supported values are:
 If a component model is given for a specific mapper via `@Mapper#componentModel()`, the value from the annotation takes precedence.
 |`default`
 
+|`mapstruct.defaultInjectionStrategy`
+| The type of the injection in mapper via parameter `uses`.
+
+Supported values are:
+
+* `field`: dependencies will be injected in fields
+* `constructor`: will be generated constructor. Dependencies will be injected via constructor.
+
+For `componentModel()=cdi` also will be generated default constructor.
```

Comment:
Perhaps reword this to "When cdi `componentModel` a default constructor will also be generated"

Reply:
Done